src/editors/components/EditorFooter/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,126 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`EditorFooter Component Save Failed, error message raised 1`] = `
+<div
+  className="editor-footer mt-auto"
+>
+  <Toast
+    action={null}
+    closeLabel="Close"
+    delay={5000}
+    show={true}
+  >
+    <FormattedMessage
+      defaultMessage="Error: Content save failed. Try again later."
+      description="Error message displayed when content fails to save."
+      id="authoring.editorfooter.save.error"
+      values={Object {}}
+    />
+  </Toast>
+  <ModalDialogFooter
+    as="div"
+  >
+    <ActionRow
+      as="div"
+      isStacked={false}
+    >
+      <ActionRowSpacer />
+      <ForwardRef
```

Comment:
I am gonna put in a quick PR to add top-level paragon mocking so this looks nicer.

Reply:
Ok That sounds great! Thanks for the swift review!